os/net/Kconfig

```diff
@@ -118,6 +118,7 @@ source "$EXTERNALDIR/tftpc/Kconfig.protocol"
 source "$EXTERNALDIR/telnetd/Kconfig.protocol"
 source "$EXTERNALDIR/smtp/Kconfig.protocol"
 
+source "$EXTERNALDIR/slsi_wifi/Kconfig"
```

Comment:
This is not a change of "apps:webclient add handshake options to connect to webbrowser".
Let's split commit.